tests/test-tokenizer-random.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, path_llama_h: str = None, path_includes: list[str] = [], path
         self.lib.llama_backend_init()
 
     def _load_libllama_cffi(self, path_llama_h: str, path_includes: list[str], path_libllama: str):
-        cmd = ["gcc", "-E", "-P", "-D__restrict=", "-D__attribute__(x)=", "-D__asm__(x)="]
+        cmd = ["gcc", "-O0", "-fno-inline", "-E", "-P", "-D__restrict=", "-D__attribute__(x)=", "-D__asm__(x)="]
```

Comment:
I think `-fno-inline` is redundant with `-O0`. And `-O0` alone works, while `-fno-inline` alone doesn't.

Anyway, I suggest resolving the conflict with `master`.